// @onelabs/dapp-kit@0.15.2/dist/esm/components/connect-modal/views/ConnectionStatus.js
// src/components/ui/Button.tsx
import { Slot } from "@radix-ui/react-slot";
import clsx from "clsx";
import { forwardRef } from "react";

// src/components/ui/Button.css.ts
import { createRuntimeFn as _7a468 } from "@vanilla-extract/recipes/createRuntimeFn";
var buttonVariants = _7a468({ defaultClassName: "Button_buttonVariants__1bruaw70", variantClassNames: { variant: { primary: "Button_buttonVariants_variant_primary__1bruaw71", outline: "Button_buttonVariants_variant_outline__1bruaw72" }, size: { md: "Button_buttonVariants_size_md__1bruaw73", lg: "Button_buttonVariants_size_lg__1bruaw74" } }, defaultVariants: { variant: "primary", size: "md" }, compoundVariants: [] });

// src/components/ui/Button.tsx
import { jsx } from "react/jsx-runtime";
var Button = forwardRef(
  ({ className, variant, size, asChild = false, ...props }, forwardedRef) => {
    const Comp = asChild ? Slot : "button";
    return /* @__PURE__ */ jsx(
      Comp,
      {
        ...props,
        className: clsx(buttonVariants({ variant, size }), className),
        ref: forwardedRef
      }
    );
  }
);
Button.displayName = "Button";

// src/components/ui/Heading.tsx
import { Slot as Slot2 } from "@radix-ui/react-slot";
import clsx2 from "clsx";
import { forwardRef as forwardRef2 } from "react";

// src/components/ui/Heading.css.ts
import { createRuntimeFn as _7a4682 } from "@vanilla-extract/recipes/createRuntimeFn";
var headingVariants = _7a4682({ defaultClassName: "Heading__1lhq1hf0", variantClassNames: { size: { sm: "Heading_headingVariants_size_sm__1lhq1hf1", md: "Heading_headingVariants_size_md__1lhq1hf2", lg: "Heading_headingVariants_size_lg__1lhq1hf3", xl: "Heading_headingVariants_size_xl__1lhq1hf4" }, weight: { normal: "Heading_headingVariants_weight_normal__1lhq1hf5", bold: "Heading_headingVariants_weight_bold__1lhq1hf6" }, truncate: { true: "Heading_headingVariants_truncate_true__1lhq1hf7" } }, defaultVariants: { size: "lg", weight: "bold" }, compoundVariants: [] });

// src/components/ui/Heading.tsx
import { jsx as jsx2 } from "react/jsx-runtime";
var Heading = forwardRef2(
  ({
    children,
    className,
    asChild = false,
    as: Tag = "h1",
    size,
    weight,
    truncate,
    ...headingProps
  }, forwardedRef) => {
    return /* @__PURE__ */ jsx2(
      Slot2,
      {
        ...headingProps,
        ref: forwardedRef,
        className: clsx2(headingVariants({ size, weight, truncate }), className),
        children: asChild ? children : /* @__PURE__ */ jsx2(Tag, { children })
      }
    );
  }
);
Heading.displayName = "Heading";

// src/components/ui/Text.tsx
import { Slot as Slot3 } from "@radix-ui/react-slot";
import clsx3 from "clsx";
import { forwardRef as forwardRef3 } from "react";

// src/components/ui/Text.css.ts
import { createRuntimeFn as _7a4683 } from "@vanilla-extract/recipes/createRuntimeFn";
var textVariants = _7a4683({ defaultClassName: "Text__h67tzv0", variantClassNames: { size: { sm: "Text_textVariants_size_sm__h67tzv1" }, weight: { normal: "Text_textVariants_weight_normal__h67tzv2", medium: "Text_textVariants_weight_medium__h67tzv3", bold: "Text_textVariants_weight_bold__h67tzv4" }, color: { muted: "Text_textVariants_color_muted__h67tzv5", danger: "Text_textVariants_color_danger__h67tzv6" }, mono: { true: "Text_textVariants_mono_true__h67tzv7" } }, defaultVariants: { size: "sm", weight: "normal" }, compoundVariants: [] });

// src/components/ui/Text.tsx
import { jsx as jsx3 } from "react/jsx-runtime";
var Text = forwardRef3(
  ({
    children,
    className,
    asChild = false,
    as: Tag = "div",
    size,
    weight,
    color,
    mono,
    ...textProps
  }, forwardedRef) => {
    return /* @__PURE__ */ jsx3(
      Slot3,
      {
        ...textProps,
        ref: forwardedRef,
        className: clsx3(textVariants({ size, weight, color, mono }), className),
        children: asChild ? children : /* @__PURE__ */ jsx3(Tag, { children })
      }
    );
  }
);
Text.displayName = "Text";

// src/components/connect-modal/views/ConnectionStatus.css.ts
var connectionStatus = "ConnectionStatus_connectionStatus__nqyit23";
var container = "ConnectionStatus_container__nqyit20";
var retryButtonContainer = "ConnectionStatus_retryButtonContainer__nqyit24";
var title = "ConnectionStatus_title__nqyit22";
var walletIcon = "ConnectionStatus_walletIcon__nqyit21";

// src/components/connect-modal/views/ConnectionStatus.tsx
import { jsx as jsx4, jsxs } from "react/jsx-runtime";
function ConnectionStatus({
  selectedWallet,
  hadConnectionError,
  onRetryConnection
}) {
  return /* @__PURE__ */ jsxs("div", { className: container, children: [
    selectedWallet.icon && /* @__PURE__ */ jsx4(
      "img",
      {
        className: walletIcon,
        src: selectedWallet.icon,
        alt: `${selectedWallet.name} logo`
      }
    ),
    /* @__PURE__ */ jsx4("div", { className: title, children: /* @__PURE__ */ jsxs(Heading, { as: "h2", size: "xl", children: [
      "Opening ",
      selectedWallet.name
    ] }) }),
    /* @__PURE__ */ jsx4("div", { className: connectionStatus, children: hadConnectionError ? /* @__PURE__ */ jsx4(Text, { color: "danger", children: "Connection failed" }) : /* @__PURE__ */ jsx4(Text, { color: "muted", children: "Confirm connection in the wallet..." }) }),
    hadConnectionError ? /* @__PURE__ */ jsx4("div", { className: retryButtonContainer, children: /* @__PURE__ */ jsx4(Button, { type: "button", variant: "outline", onClick: () => onRetryConnection(selectedWallet), children: "Retry Connection" }) }) : null
  ] });
}
export {
  ConnectionStatus
};
