// @onelabs/dapp-kit@0.15.2/dist/cjs/components/connect-modal/ConnectModal.css.js
"use strict";
var __defProp = Object.defineProperty;
var __getOwnPropDesc = Object.getOwnPropertyDescriptor;
var __getOwnPropNames = Object.getOwnPropertyNames;
var __hasOwnProp = Object.prototype.hasOwnProperty;
var __export = (target, all) => {
  for (var name in all)
    __defProp(target, name, { get: all[name], enumerable: true });
};
var __copyProps = (to, from, except, desc) => {
  if (from && typeof from === "object" || typeof from === "function") {
    for (let key of __getOwnPropNames(from))
      if (!__hasOwnProp.call(to, key) && key !== except)
        __defProp(to, key, { get: () => from[key], enumerable: !(desc = __getOwnPropDesc(from, key)) || desc.enumerable });
  }
  return to;
};
var __toCommonJS = (mod) => __copyProps(__defProp({}, "__esModule", { value: true }), mod);

// src/components/connect-modal/ConnectModal.css.ts
var ConnectModal_css_exports = {};
__export(ConnectModal_css_exports, {
  backButtonContainer: () => backButtonContainer,
  closeButtonContainer: () => closeButtonContainer,
  content: () => content,
  overlay: () => overlay,
  selectedViewContainer: () => selectedViewContainer,
  title: () => title,
  viewContainer: () => viewContainer,
  walletListContainer: () => walletListContainer,
  walletListContainerWithViewSelected: () => walletListContainerWithViewSelected,
  walletListContent: () => walletListContent,
  whatIsAWalletButton: () => whatIsAWalletButton
});
module.exports = __toCommonJS(ConnectModal_css_exports);
var backButtonContainer = "ConnectModal_backButtonContainer__f5yvs56";
var closeButtonContainer = "ConnectModal_closeButtonContainer__f5yvs57";
var content = "ConnectModal_content__f5yvs52";
var overlay = "ConnectModal_overlay__f5yvs50";
var selectedViewContainer = "ConnectModal_selectedViewContainer__f5yvs55";
var title = "ConnectModal_title__f5yvs51";
var viewContainer = "ConnectModal_viewContainer__f5yvs54";
var walletListContainer = "ConnectModal_walletListContainer__f5yvs59";
var walletListContainerWithViewSelected = "ConnectModal_walletListContainerWithViewSelected__f5yvs5a";
var walletListContent = "ConnectModal_walletListContent__f5yvs58";
var whatIsAWalletButton = "ConnectModal_whatIsAWalletButton__f5yvs53";
